### PR TITLE
[BugBash] SC2001: Remove sed as it's not required

### PR DESCRIPTION
**Pull request by anuragpaliwal80**
### What's changed and how it works?
Remove sed as it's not required

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

CHANGELOG

- [ ] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by anuragpaliwal80** [comment]
Markdown check are failing as

```
ERROR: 1 dead links found!
[✖] https://www.microsoft.com/ → Status: 0
```

File: install.sh [hunk 1]
```diff
@@ -318,7 +318,7 @@ install_kubectl() {
     err_msg=$(kubectl version --client=true 2>&1 1>/dev/null)
     if [ "$err_msg" == "" ]; then
         v=$(kubectl version --client=true | sed 's/.*GitVersion:\"v\([0-9.]*\).*/\1/g')
-        target_version=$(echo "${kubectl_version}" | sed s/v//g)
+        target_version=${kind_version//v/g}
```
Comment (Andrewmatilde) [inline install.sh:321]:
```
#!/usr/bin/env bash

kubectl_version="v0.11.1"
target_version=$(echo "${kubectl_version}" | sed s/v//g)
echo ${target_version}
target_version=${kubectl_version//v/g}
echo ${target_version}
```
I have write a test script , but it failed with:./test.sh: 6: ./test.sh: Bad substitution

Comment (anuragpaliwal80) [reply install.sh]: #!/usr/bin/env bash

```
kubectl_version="v0.11.1"
target_version=$(echo "${kubectl_version}" | sed s/v//g)
echo ${target_version}
target_version=${kubectl_version//v}
echo ${target_version}
```
Returns ->
```
0.11.1
0.11.1
```

**Review by STRRL — APPROVED** [review]
LGTM!

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Pull request merged**